docs/reference/islandora_defaults_reference.md

```diff
@@ -0,0 +1,395 @@
+# Drupal Configuration Provided By Islandora Defaults
+
+This is a list of the configuration items provided by the Islandora Defaults module. 
+
+## Entities
+
+(see Glossary: [Entity](../user-documentation/glossary.md#entity))
+
+### Content Types (i.e. Node bundles)
+
+|Name  					|Description  		 		|Comments							|
+|---					|---					|---								|
+|Repository Item (islandora_object)   	|An item in your Islandora repository.  |Defaults provides only one node type to use with Islandora, and different behaviours are triggered by different field values. If you would like to use another content type with Islandora, it will need Islandora behaviours configured appropriately (see below). 	|
+
+
+### Media Types (i.e. Media bundles)
+
+|Name  					|Description  		 		|Comments							|
+|---					|---					|---								|
+|Audio (audio)   	| A locally hosted audio file.  	| This is Drupal's built-in Audio media type. Islandora Defaults overwrites the field storage so that it is not actually a locally hosted file. The description is misleading.	|
```

Comment:
True, but instead of documenting how it's misleading here, we should update the description as a follow up task so we no longer have to point that out.

Reply:
Addressed by https://github.com/Islandora/islandora_defaults/pull/45